pulp_smash/tests/pulp3/utils.py

```diff
@@ -262,18 +262,19 @@ def delete_orphans(cfg=None):
     api.Client(cfg, api.safe_handler).delete(ORPHANS_PATH)
 
 
-def get_repo_versions(repo):
-    """Return hrefs of repository versions.
+def get_version_hrefs(repo):
+    """Return hrefs for repository versions.
 
     :param repo: A dict of information about the repository.
     :returns: A sorted list with the hrefs of repository versions.
     """
     client = api.Client(config.get_config(), api.json_handler)
-    versions = client.get(repo['_versions_href'])['results']
-    return sorted(
-        [version['_href'] for version in versions],
-        key=lambda url: int(urlsplit(url).path.split('/')[-2])
-    )
+    attributes = [
+        version_href['_href']
+        for version_href in client.get(repo['_versions_href'])['results']
+    ]
+    attributes.sort(key=lambda url: int(urlsplit(url).path.split('/')[-2]))
+    return attributes
```

Comment:
+1 Sorting in-place takes about the same amount of time, while using less memory. This avoids unnecessary cache line evictions. (I say this while also recognizing that performance is less important than making code work or making code correct.)